server/app.mjs

```diff
@@ -140,6 +140,10 @@ const mount = async (
     workspace: scriptWorkspace,
     prompt: true,
     confirm: true,
+    env: [
+      ...Object.entries(process.env).map(([key, value]) => `${key}=${value}`),
```

Comment:
This shouldn't be necessary. Add env vars here should append to the existing environment.

Reply:
I looked at the code and it doesn't seem so. https://github.com/gptscript-ai/node-gptscript/blob/main/src/gptscript.ts#L88

Reply:
It happens elsewhere.